### PR TITLE
CONTRIBUTING: clarify guideline for new PRs

### PR DESCRIPTION
**Pull request by mfrw**
PRs should be created against the `main` branch as base instead of
`master`

Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Closes: #7449

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: CONTRIBUTING.md
Comment (blueskyson) [inline CONTRIBUTING.md:26]: I think the description can be refined to the following:
```suggestion
*Note*: please base your PR against the `main` branch, and check if there's already a pull request in progress for it.
```

**Review by blueskyson — APPROVED** [review]

**Comment by mfrw** [comment]
Probably can anyone request changes here to block merging, till we have signoff from other folks as well.
@navarroaxel ? maybe ? can you please block this :)

**Review by navarroaxel — CHANGES_REQUESTED** [review]
sure

**Review by marchersimon — APPROVED** [review]
Thanks @mfrw

**Review by CleanMachine1 — APPROVED** [review]
the word base could confuse some newcomers however LGTM

**Comment by marchersimon** [comment]
What just happened there? Is this some kind of GitHub superpower I haven't heard of yet?
image

**Comment by navarroaxel** [comment]
@marchersimon he removed my block to merge :laughing:

**Comment by CleanMachine1** [comment]
@marchersimon 

Cant tell which part of what you mean so heres both just in case :shrug: 

I removed the review from pressing "changes requested" in the merge section of this page, then the "..."  and dismissed.

The "merge block" is simply a review where this is selected

```
Request changes

Submit feedback that must be addressed before merging.
```

Not really a "merge block" however its a good alternative!

**Comment by marchersimon** [comment]
I didn't know it was possible to dismiss requested changes 😅 
And I also didn't know about merge blocks. Thanks for the explanation.

**Review by sbrl — APPROVED** [review]
Seems good to me.

**Comment by mfrw** [comment]
Thank you all. Merging now :)

**Pull request merged**
